content/blog/posts/2022-07-08-go-serverless-functions-with-ember-and-sst-on-aws.md

```diff
@@ -0,0 +1,200 @@
+
+## Connecting the Wires
+
+After generating the frontend application we'll make some adjustments to the `./stacks/MyStack.ts` file. Go ahead and add an import for `StaticSite` and let's also add a new StaticSite that will point to our Ember application:
+
+```javascript
+// we added the StaticSite import here
+import { StackContext, Api, StaticSite } from "@serverless-stack/resources";
+
+export function MyStack({ stack }: StackContext) {
+  const api = new Api(stack, "api", {
+    routes: {
+      "GET /": "functions/lambda/main.go",
+    },
+  });
+
+  // we added instantiation of StaticSite for the Ember application here
+  const site = new StaticSite(this, "EmberSite", {
+    path: "frontend",
+    buildOutput: "dist",
+    buildCommand: "npm run build",
+    environment: {
+      // Pass in the API endpoint to our app
+      EMBER_APP_API_URL: api.url,
+    },
+  });
+
+  stack.addOutputs({
+    ApiEndpoint: api.url,
+  });
+}
+```
+
+This will tell your SST project about your Ember application, how to build the Ember code and where to save the build. After this, we should go ahead and run `npm install` or `yarn` in both the SST project root and the Ember project root. Note, at the time of this writing, we had to use Node 16 and chose the latest LTS, `lts/gallium`, but your mileage may vary.
+
+If you now run `yarn start` or `npm start` from the SST project root this will start the SST side of things, which includes creating debug deployments that help run the serverless function locally. The SST CLI will ask for an environment name which we'll provide as `dev`, but you can call it whatever you like.
+
+This may take several minutes to complete since SST is conducting an actual initial deployment (SST uses something they call [live lambda development](https://docs.sst.dev/live-lambda-development), which includes deploying services for your development environment to interact with), but at the end of the process you should see something like the following which should include the `ApiEndpoint` where the serverless function can be reached:
+
+![SST server started and provided API endpoint url](/img/blog/serverlesss-functions-direct-to-aws-with-ember/sst-start-outputs-server-url.png)
+
+Now before booting our Ember app we'll probably want to set things up so that the Ember application communicates with the serverless function. We'll do that by generating an application route in the Ember app and sending a fetch request to the `ApiEndpoint` URL as a gut check to ensure we can retrieve some data.
```

Comment:
~Now~ before booting our Ember app we'll ~probably~ want to set things up so that the ~Ember application~ frontend(?) communicates with the serverless function. We'll do that by generating an application route in the Ember app and sending a fetch request to the `ApiEndpoint` URL as a gut-check to ensure we can retrieve some data.